client_updater.go

```diff
@@ -113,7 +113,6 @@ var nosaveMessages = map[string]struct{}{
 // saveState stores server configuration to the standard config file.
 func saveState(lastMessages map[string]interface{}) {
 
-	lastMessages["CURRENTTIME"] = time.Now().Format(time.UnixDate)
```

Comment:
Is this what fixed the CURRENTTIME problem in #74? I liked having current time in the config.yaml file. I'd rather you didn't take it out.

Can you add it back in and see whether #74 recurs?

Reply:
I added it back in, it doesn't send recreate the problem. I'll leave it in.